kubernetes/deployment-staging-azure-canary.tmpl

```diff
@@ -0,0 +1,171 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: panoptes-staging-azure-canary-app
```

Comment:
I'm wondering if we should use the same names as existing deployments but instead namespace the canary deployment? I believe using a namespace will help us cleanup the canary deploy...thoughts?

Reply:
We don't have very many namespaces and I'm not sure if there are any good reasons to separate them. I don't see that it would make the deployment easier to remove, since there are no other moving parts or networking or w/e.